### DISCUSSION TITLE
Nextcloud log when querying for floccus lockfile

### DISCUSSION THREAD

**Discussion by SlaveOfCats** [floccusaddon/floccus#1261] [Q&A] [answered]
I am using floccus 4.17.1 with Firefox 104.0.2, and I'm using Nextcloud 24.0.5 as a WebDAV server.

I'm seeing in the Nextcloud log files that an event is raised every time floccus syncs. It appears that Nextcloud doesn't like it when floccus queries for the lockfile and the lockfile isn't present (which is the normal case for floccus).

Is there a way to prevent this? It's not a serious issue, but it's a little annoying to have the log files filling up with benign events.

Thanks.

**Comment by marcelklehr** [comment]
Can you post the log file entry from the nextcloud log?

**Reply by SlaveOfCats** [reply, marked as answer]
Since posting this question, I have updated to Firefox 119, floccus 4.19.1, and NextCloud 26, and I am no longer seeing the problem.
